networkx/generators/tests/test_line.py

```diff
@@ -174,6 +174,13 @@ def test_claw(self):
         G.add_edges_from(G_edges)
         pytest.raises(nx.NetworkXError, nx.inverse_line_graph, G)
 
+    def test_edgeless_graph(self):
+        # This graph has 2 nodes and 0 edges
+        G = nx.Graph()
+        G_nodes = [0, 1]
+        G.add_nodes_from(G_nodes)
+        pytest.raises(nx.NetworkXError, nx.inverse_line_graph, G)
```

Comment:
Just another minor suggestion - using `pytest.raises` as a context manager is generally more readable (gets rid of signature forwarding). We can also match the exception message to ensure we're hitting the expected exception branch

```suggestion
        with pytest.raises(nx.NetworkXError, match=".*edgeless graph"):
            nx.inverse_line_graph(G)
```

Reply:
yup I agree, thanks for the feedback!